bundles/org.openhab.core.io.websocket/pom.xml

```diff
@@ -20,6 +20,11 @@
       <artifactId>org.openhab.core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core.io.rest.auth</artifactId>
```

Comment:
This dependency brings you a lot of stuff, making websocket part embedding a lot of jaxrs dependencies. From this point of view, if you collapse dependencies you could try to do websockets at jaxrs level. it will be slower, but will keep things consistent.